main.py

```diff
@@ -36,3 +36,5 @@
     process_data(subject_qualisys_json_path, qualisys_df, pupil_df)
```

Comment:
separate "process_data" from "plot_data" functionalities

that is - 

First - Do all the processing and save the data out in whatever kinda way (i.e. in `process_data` function), 

THEN - Have your plotting method load in that data and try to make a visualization out of it

Reply:
separated ✔️